python/ray/tests/test_object_spilling_3.py

```diff
@@ -363,6 +364,33 @@ def check(x):
         ray.get(ref)
 
 
+def test_evict_secondary_before_spill(ray_start_cluster, object_spilling_config):
```

Comment:
nit: add a docstring to the test

"""
Verify spilling is not triggered when secondary copies are evictable
"""

Reply:
I think this is fine because the test name already says this.